notesapplication/notes/api_v1/serializers.py

```diff
@@ -1,8 +1,36 @@
-from django.contrib.auth.password_validation import validate_password
+from datetime import date
+
 from rest_framework import serializers
-from rest_framework.validators import UniqueValidator
+from user.models import Users
+
+from ..models import Comment
 from ..models import Note
-from datetime import date
+from ..models import NoteVersion
+
+
+class CommentSerializer(serializers.ModelSerializer):
+    """
+    CommentSerializer : Serializer for comment class and CRUD operations of comments
+    """
+
+    class Meta:
+        model = Comment
+        fields = "__all__"
+        read_only_fields = ("user",)
+
+    def create(self, validated_data):
+        users = Users.objects.all()
+        request = self.context.get("request", None)
+        if request.user in users:
```

Comment:
I think there's no need to check users in DB. The fact that you have `request.user` indicates that the valid user exists. Therefore, something like `if request.user` would suffice.